modules/installation-osp-setting-cloud-provider-options.adoc

```diff
@@ -39,17 +39,30 @@ Configuring Octavia for load balancing is a common case. For example:
 [LoadBalancer]
 use-octavia=true <1>
 lb-provider = "amphora" <2>
-floating-network-id="d3deb660-4190-40a3-91f1-37326fe6ec4a"<3>
+floating-network-id="d3deb660-4190-40a3-91f1-37326fe6ec4a" <3>
+create-monitor = True <4>
+monitor-delay = 10s <5>
+monitor-timeout = 10s <6>
+monitor-max-retries = 1 <7>
 #...
 ----
 <1> This property enables Octavia integration.
 <2> This property sets the Octavia provider that your load balancer uses. It accepts `"ovn"` or `"amphora"` as values. If you choose to use OVN, you must also set `lb-method` to `SOURCE_IP_PORT`.
 <3> This property is required if you want to use multiple external networks with your cluster. The cloud provider creates floating IP addresses on the network that is specified here.
+<4> Setting this property to `True` causes the cloud provider to create health monitors for the Octavia load balancers. As of {rh-openstack} 16.1 and 16.2 this is only supported for Amphora provider.
+<5> This property is required when `create-monitor` is set to `True`. It sets how often endpoints are monitored in `time.ParseDuration()` format.
+<6> This property is required when `create-monitor` is set to `True`. It sets how long monitoring requests time out in `time.ParseDuration()` format.
```

Comment:
Not sure this is grammatically correct.
```suggestion
<6> This property is required when `create-monitor` is set to `True`. It sets how long before monitoring requests time out in `time.ParseDuration()` format.
```

Reply:
Best way to ensure grammatical correctness is to start from a rudimentary but painfully wordy statement and then whittle it down. 

How would you explain what is happening here to someone who's coming into this cold?